main.go

```diff
@@ -33,61 +34,144 @@ import (
 	// +kubebuilder:scaffold:imports
 )
 
+const (
```

Comment:
Minor comment. Can you create 2 constant blocks - one for defaults and another for flag names?
Another minor comment (this is obviously a matter of personal choice) - I would prefer `flag` as a suffix rather than as a prefix. But this you can accept or reject. Whatever convention we follow then lets stick to that convention throughout.

Reply:
Sure Madhav, will have two separate constant blocks. But naming-wise, I would prefer to keep it the same since the prefix `flag` or `default` denotes the purpose of the constant. So all `flag`-prefixed constants denote flag names, while all `default`-prefixed constants denote default values for the flags. Hope that's fine.